web/modules/custom/joinup_migrate/joinup_migrate.post_update.php

```diff
@@ -159,3 +161,63 @@ function joinup_migrate_post_update_more_redirects() {
     }
   }
 }
+
+/**
+ * Switches the ownership of the 'Sharing and reuse of IT solutions' collection.
+ */
+function joinup_migrate_post_update_srits_change_owner() {
+  $collection_id = 'http://data.europa.eu/w21/e5f0febe-f29c-428a-a1a2-6bb0ccd37949';
+  // The 'Joinup moderator'.
+  $current_owner_id = '700003';
+  // The 'Joinup editor'.
+  $new_owner_id = '6363';
+  /** @var \Drupal\og\MembershipManagerInterface $membership_manager */
+  $membership_manager = \Drupal::service('og.membership_manager');
+  /** @var \Drupal\Core\Entity\EntityTypeManager $entity_storage */
+  $entity_type_manager = \Drupal::service('entity_type.manager');
+
+  $collection = $entity_type_manager->getStorage('rdf_entity')->load($collection_id);
+  if (empty($collection)) {
+    throw new Exception(t('Collection with id :id was not found.', [
+      ':id' => $collection_id,
+    ]));
+  }
+
+  $current_owner = $entity_type_manager->getStorage('user')->load($current_owner_id);
+  if (empty($current_owner)) {
+    throw new Exception(t('User with id :id was not found.', [
+      ':id' => $current_owner_id,
+    ]));
+  }
+
+  $new_owner = $entity_type_manager->getStorage('user')->load($new_owner_id);
+  if (empty($new_owner)) {
+    throw new Exception(t('User with id :id was not found.', [
+      ':id' => $new_owner_id,
+    ]));
+  }
+
+  // Revoke the administrator role from the 'Joinup moderator'.
+  $membership = $membership_manager->getMembership($collection, $current_owner);
+  if (!empty($membership)) {
+    $membership->revokeRole(OgRole::getRole('rdf_entity', 'collection', OgRoleInterface::ADMINISTRATOR));
```

Comment:
You can leave its as it is. For the future: we have `$membership->revokeRoleById()` :)